src/scheduler.h

```diff
@@ -55,6 +55,11 @@ class CScheduler
     // need more accurate scheduling, don't use this method.
     void scheduleEvery(Function f, int64_t deltaMilliSeconds);
 
+    // Mock the scheduler to fast forward in time.
```

Comment:
style-nit in commit 3f5cebe47b:

Using `/**` will make this a doxygen comment. I know that the other functions don't use doxygen comment, but for new code we should aim for that.